ranger/core/actions.py

```diff
@@ -1651,3 +1651,30 @@ def rename(self, src, dest):
             self.notify(err)
             return False
         return True
+
+    def display_macro(self, name):
+        # Get the macro's value.
+        if name[0] != ranger.MACRO_DELIMITER:
+            self.notify('Invalid macro name: %s' % name)
+            return False
+        macros = self.get_macros()
+        try:
+            value = macros[name[1:]]
+            # Macro value returned by get_macros() can be:
+            # - String: if the macro contains a single file
+            # - List: if the macro contains multiple files
+            # - MACRO_FAIL: if the macro hasn't been set
+            if value == MACRO_FAIL:
+                value = []
+            if isinstance(value, str):
+                value = [value]
+        except KeyError:
+            self.notify('Invalid macro name: %s' % name)
```

Comment:
Extract these two formatting strings into a variable. Less room for error.

Reply:
Done.